common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/OpenIdProviderConfigurationClient.java

```diff
@@ -139,9 +141,9 @@ public synchronized OpenIdProviderConfiguration loadOpenIdProviderConfiguration(
                     "Using request URL: " + configUrl
             );
 
-            // Suppressing deprecation warnings due to the deprecated method HttpRequest.sendGet(). Raised issue https://github.com/AzureAD/microsoft-authentication-library-common-for-android/issues/1038
-            @SuppressWarnings(WarningType.deprecation_warning)
-            final HttpResponse providerConfigResponse = HttpRequest.sendGet(configUrl, new HashMap<String, String>());
+            final Map<String, String> headers = new HashMap<>();
+
+            final HttpResponse providerConfigResponse = httpClient.get(configUrl, headers);
```

Comment:
If we don't do anything with the header reference, can we just pass a new HashMap to the get call?